pkg/controller/account/ec2.go

```diff
@@ -46,12 +46,23 @@ func (r *ReconcileAccount) InitializeSupportedRegions(reqLogger logr.Logger, acc
 	vCPUQuota, _ := r.getDesiredVCPUValue(reqLogger)
 	reqLogger.Info("retrieved desired vCPU quota value from configMap", "quota.vcpu", vCPUQuota)
 
+	// FIXME: Is this the right place to get it - in the reconcile loop for accounts there are two call to this initialize, so should reduce weird copy-paste errors in the future.
```

Comment:
I don't like seeing FIXME in docstrings, we should ideally track this in a ticket elsewhere, or look at resolving it before the PR is merged

asyncRegionInit is only called in initializeRegions which hits 2 flows, but they never overlap, so I don't think we'll be calling this twice in a loop

Reply:
Oh yea that was more like a note for reviewers, if this seems to be something we would like to do elsewhere, so I think I would just remove the FIXME + comment.